build/assets.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GIT_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+for f in $GIT_ROOT/pages/assets/**/*
+do
```

Comment:
nit: our bash style is to put do / then on the previous line with a semicolon (e.g. `for f in $FOO; do`).
Same for `if` below.